docs/maui/views/popup/popup-options.md

```diff
@@ -0,0 +1,139 @@
+
+```csharp
+await ShowPopupAsync<SimplePopup>(Shell.Current, new PopupOptions
+{
+    CanBeDismissedByTappingOutsideOfPopup = false
+});
+```
+
+> [!NOTE]
+> It is possible to cache `PopupOptions` to reuse on future navigation actions or with other popups that share the same configuration, that would prevent unnecessary allocations.
+
+## Customize the Overlay Color
+
+`PopupOptions` provides the `PageOverlayColor` property which can control the `Color` that overlays the current page. The following example shows how to set the `PageOverlayColor` to orange.
+
+```csharp
+await ShowPopupAsync<SimplePopup>(Shell.Current, new PopupOptions
+{
+    PageOverlayColor = Colors.Orange
+});
+```
+
+![Popup page overlay](../../images/views/popup/popup-page-overlay-opaque.png "Popup rendering with an opaque orange background")
+
+The above `Popup` renders with an opaque background, to control the opacity modify the alpha property of a `Color`. The following example shows how to
+
+```csharp
+await ShowPopupAsync<SimplePopup>(Shell.Current, new PopupOptions
+{
+    PageOverlayColor = Colors.Orange.WithAlpha(0.5f)
+});
+```
+
+![Popup page transparent overlay](../../images/views/popup/popup-page-overlay-translucent.png "Popup rendering with a translucent orange background")
+
+## Customize the Popup Border
+
+`PopupOptions` provides the `Shape` property which can control the appearance of the border around the content displayed in the `Popup`. The following example shows how to set the border to be a rounded rectangle with a corner radius of 4 and a stroke of blue.
+
+```csharp
+await ShowPopupAsync<SimplePopup>(Shell.Current, new PopupOptions
+{
+    Shape = new RoundRectangle
+    {
+        CornerRadius = new CornerRadius(4),
+        Stroke = Colors.Blue,
+        StrokeThickness = 4
+    }
+});
+```
+
+![Popup blue border stroke](../../images/views/popup/popup-blue-border.png "Popup rendering with a blue border")
+
+For more details on how to customize the `Shape` property see [.NET MAUI Shapes](/dotnet/maui/user-interface/controls/shapes/).
+
+### Disable the Popup Border
+
+In order to disable the border on a Popup, simply set the `Shape` property to `null`. The following example shows how to achieve this
+
+```csharp
+await ShowPopupAsync<SimplePopup>(Shell.Current, new PopupOptions
+{
+    Shape = null
+});
+```
+
+![Popup no border](../../images/views/popup/popup-no-border.png "Popup rendering with no border")
+
+## Customize the Popup Shadow
+
+`PopupOptions` provides the `Shadow` property which can control the shadow which is applied to the `Popup`. The following example shows how to set the shadow to be green with an opacity of 80%.
+
+```csharp
+await ShowPopupAsync<SimplePopup>(Shell.Current, new PopupOptions
```

Comment:
Let's avoid using `PopupService` here to keep these docs simple and to avoid implying devs need to use `PopupService` in order to use `PopupOptions`

```suggestion
await this.ShowPopupAsync(new SimplePopup(), new PopupOptions
```